readers/cassandra/messages.go

```diff
@@ -114,3 +137,33 @@ func buildCountQuery(chanID string, names []string) string {
 
 	return fmt.Sprintf(cql, condCQL)
 }
+
+func parseFlat(flat interface{}) interface{} {
```

Comment:
Looks like this function is duplicated for each reader. Should we maybe export it?

Reply:
Yes, I'll probably put it in the `transformers/json` as a utility function.

Reply:
Maybe `transformers/pkg` as shared package? Its oft to have multiple `pkg` and `internal` (not just in root).

Reply:
I put it in `pkg/tranformers/json`. It's a utility function used by JSON transformer and readers, so I guess it's OK to leave it there.